REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.3
 MathProgBase 0.3.8
 ECOS 0.4.1
+FactCheck 0.2.5
```

Comment:
Oops, I guess we never tidied that up. @IainNZ 
If you just use ``Pkg.test("Convex", coverage=true)`` then first the testing dependencies will be installed and then ``test/runtests.jl`` will be run. That's the "right" way to do it.